gsdk/src/client.rs

```diff
@@ -190,7 +186,21 @@ impl Rpc {
         method: &str,
         params: RpcParams,
     ) -> Result<Res> {
-        self.rpc.request(method, params).await.map_err(Into::into)
+        let mut retries = 0;
+
+        loop {
+            let r = self
+                .rpc
+                .request(method, params.clone())
+                .await
+                .map_err(Into::into);
+
+            if retries == self.retries || r.is_ok() {
```

Comment:
perhaps it's worth to log an error?

Reply:
processed warnings instead of errors since the last try will emit error anyway